### DISCUSSION TITLE
Enable guests to purchase unlimited number of On Ride Photos

### DISCUSSION THREAD

**Discussion by astraylife1** [OpenRCT2/OpenRCT2#17065] [Feature requests]
Currently guests may only purchase one on-ride photo for each of the following ride types (steel coasters, wooden coaster, water ride, inverted/flying/4D coaster). It would be nice if they could purchase a photo for every ride that offers a photo. Would make it more worthwhile to add photo sections to all coasters/rides.

**Comment by duncanspumpkin** [comment]
This would be really quite hard to implement for little benefit. I feel the way peep items work should in general should be changed though (atm it's based on a flag in a 64bit field and a bunch of other fields to handle extra parameters). Perhaps making it so a peep can only ever hold X quantity of items (where x is a small number like 8) and then allow for replacing the oldest one when purchasing a new one.

**Reply by astraylife1** [reply]
I didn't realize it would be so complicated vs just upping the limit of photos per ride type or something.

As it stands, peeps can only hold 12 items in the game: map, umbrella, sunglasses, shirt, hat, toy, balloon, food, and four photo types. The balloon and food are the only items that can be repurchased. But it would be nice if they could buy more of certain items like toys, photos, or shirts. Just trying to expand the revenue streams in my parks. Then it would make more sense to have souvenir stands at the major attraction exits. As of now, I can only really do that with balloons.
